### ISSUE TITLE
Provide issues for each "fix" with description

### ISSUE THREAD

**Issue by evil-shrike** [6pac/SlickGrid#48]
Hi.
The repo is a good idea. But I beleive its current state is unacceptable for being a replacement of the master SlickGrid. 
The fix list contains just short notes like "fix bug with refresh last row of grid". What does it mean? What bug with refresh? 
There should an issue with full description of the bug. And so for each fix. Then it will make sense.

**Comment by 6pac** [comment]
In most cases I reference the issue number where a reported issue has been fixed. However if I discover a problem, as I did with that one, I don't usually raise an issue. I have been thinking it might be good practice to do so, but it could be a lot more work too. I'll review fixes to see the state of them.
Whenever a new feature is added, there is nearly always a new example, but I haven't gone back and updated the original docs. Again, on the to-do list, but a big task. 
If anyone else is willing to systematically go through the commits and update the docs, that would be a big help. The commits are at least small and specific.
